src/pb/FDoper.h

```diff
@@ -120,6 +120,7 @@ class FDoper : public FDoperInterface
             delete lower_order_grid_;
             lower_order_grid_ = nullptr;
         }
+        del2_4th_tm_.print(std::cout);
```

Comment:
This call should not be here. WE print timers together at the end of run.

Reply:
I will delete this line in the next commit.